src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/jpa/repository/PrisonIncentiveLevelRepository.kt

```diff
@@ -74,16 +73,16 @@ interface PrisonIncentiveLevelRepository : CoroutineCrudRepository<PrisonIncenti
    * Sets levels other than the one privided to be non-default for this prison
    * Used to ensure there is only one default level for admission in a prison
    */
-  @Modifying
```

Comment:
why no longer `@Modifying`?

Reply:
it's a slightly strange annotation: it tells spring to only care about the number or rows changed so the `… RETURNING ???` is ignored if you use the annotation.

Reply:
Weird, oh I see it means "this is an update query, ignore the returned value" 👌